homeassistant/components/yolink/number.py

```diff
@@ -0,0 +1,128 @@
+"""YoLink device number type config settings."""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from yolink.client_request import ClientRequest
+from yolink.const import ATTR_DEVICE_SPEAKER_HUB
+from yolink.device import YoLinkDevice
+
+from homeassistant.components.number import (
+    NumberEntity,
+    NumberEntityDescription,
+    NumberMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import YoLinkCoordinator
+from .entity import YoLinkEntity
+
+OPTIONS_VALUME = "options_volume"
+
+
+@dataclass
+class YoLinkNumberTypeConfigEntityDescriptionMixin:
+    """Mixin for device type."""
+
+    exists_fn: Callable[[YoLinkDevice], bool] = lambda _: True
+
+
+@dataclass
```

Comment:
There's a recent blogpost about entity descriptions please check it out

Also. Look in the code for kw_only. It can make you remove the mixin

Reply:
i will create a new PR for this